runelite-client/src/main/resources/net/runelite/client/plugins/roofremoval/overrides.jsonc

```
@@ -724,6 +724,14 @@
     }
   ],
   "10288": [ // Yanille East
+    {
+      "rx1": 27,
+      "ry1": 12,
+      "rx2": 34,
```

This doesn't need to extend out so far; the values are inclusive of both edges
```suggestion
      "rx2": 33,
```